m3-perf-testing-app/app/serializers/application.js

```diff
@@ -0,0 +1,18 @@
+import { assign } from '@ember/polyfills';
+
+export default class ApplicationSerializer {
+  constructor(createArgs) {
+    assign(this, createArgs);
+  }
```

Comment:
We might not need to use `@ember/polyfills` for this. I think we can reasonably decide not to support IE11 in our test app.

```suggestion
export default class ApplicationSerializer {
  constructor(createArgs) {
    Object.assign(this, createArgs);
  }
```